.github/workflows/rubocop.yml

```diff
@@ -0,0 +1,19 @@
+name: RuboCop
+on: [pull_request]
+jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+      - name: Setup
+        run: |
+          gem install bundler
+          bundle config set without assets console development dynflow_sidekiq ec2 gce journald jsonp libvirt openid openstack ovirt redis service telemetry test vmware
+          bundle config set path vendor/bundle
+          bundle install --jobs=3 --retry=3
+      - name: Run rubocop
+        run: bundle exec rake rubocop
```

Comment:
Is there a particular reason why you don't use the `actions/cache@` step? It makes everything much faster.

Reply:
I've used `bundler-cache`, that we've discovered recently in https://github.com/theforeman/foreman_ansible/pull/368 and works better then manual usage of `actions/cache@`